sdk/rack/process.go

```diff
@@ -114,12 +111,13 @@ func (c *Client) ProcessRun(app string, opts types.ProcessRunOptions) (int, erro
 	defer r.Close()
 
 	var code int
+	var rerr string
 
-	if err := helpers.Stream(helpers.CodeGrabber(opts.Output, &code), r); err != nil {
+	if err := helpers.Stream(helpers.CodeGrabber(opts.Output, &code, &rerr), r); err != nil {
 		return 0, err
```

Comment:
Maybe we can. I do like 71. But I believe this should be set in the backend provider (aws, local, etc). I'll take a look. I want to prevent littering 71s everywhere. Unless I misunderstood.